labgrid/driver/sshdriver.py

```diff
@@ -48,6 +49,9 @@ def on_activate(self):
         if not self.networkservice.password:
             self.ssh_prefix += ["-o", "PasswordAuthentication=no"]
 
+        if self.options:
+            self.ssh_prefix += self.options.split(" ")
```

Comment:
https://docs.python.org/3/library/shlex.html#shlex.split is probably better in this case.

As some special cases around quoting are not easily handled even with `shlex.split`, we should support optionally having a list instead of a string. In that case, they could simply be passed along.